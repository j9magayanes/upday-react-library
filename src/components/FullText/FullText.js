import React from 'react';
import './fullText.css';


export const FullText = ({ title, fulltext }) => {
  return (
    <h2 id="fulltext" title={title}>{fulltext}</h2>
  );
};