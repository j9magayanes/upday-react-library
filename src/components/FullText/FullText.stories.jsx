
import React from 'react';

import { FullText } from './FullText';


export default {
  title: 'Form/FullText',
  component: FullText,
};

const Template = (args) =><FullText   {...args} />;

export const Meinestadt = Template.bind({});
Meinestadt.args = {
  label: 'meinestadt',
  title: 'meinestadt',
  fulltext: 'Mehr als',

};

export const Web = Template.bind({});
Web.args = {
  title: 'Web',
  size: 'web',
  fulltext: 'Mehr als'
};