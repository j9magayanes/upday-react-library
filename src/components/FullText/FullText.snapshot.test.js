import React from 'react';
import renderer from 'react-test-renderer';
import { FullText } from './FullText';

test('renders FullText component correctly', () => {
  const props = {
    title: 'Example Title',
    fulltext: 'This is the full text content.',
  };

  const component = renderer.create(<FullText {...props} />);
  const tree = component.toJSON();

  expect(tree).toMatchSnapshot();
});
