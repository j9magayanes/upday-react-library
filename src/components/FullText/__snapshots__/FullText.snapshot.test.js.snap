// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders FullText component correctly 1`] = `
<h2
  id="fulltext"
  title="Example Title"
>
  This is the full text content.
</h2>
`;
